src/app/sessions/components/sessions-propositions-form/sessions-propositions-form.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { Judge } from '../../../judges/models/judge.model';
+import { Room } from '../../../rooms/models/room.model';
+import { SessionPropositionQuery } from '../../models/session-proposition-query.model';
+import * as moment from 'moment'
+
+@Component({
+    selector: 'app-sessions-propositions-form',
+    templateUrl: './sessions-propositions-form.component.html',
+    styleUrls: ['./sessions-propositions-form.component.scss']
+})
+export class SessionsPropositionsFormComponent implements OnInit {
+
+    searchParams = {
+        from: new Date(),
+        to: new Date(),
+        durationInMinutes: 1,
+        roomId: null,
+        judgeId: null
+    } as SessionPropositionQuery;
```

Comment:
(Curiosity time) Initial values for roomid and judgeId are null. What will the backend respond if we click 'Search' with these default parameters?

Reply:
API is prepared to not to add them and also to not to receive them (are optional)